pkg/ingest/ingest.go

```diff
@@ -51,9 +51,10 @@ func sendRequest(iType IngestType, client *http.Client, lines []byte, url string
 	var requestStr string
 	switch iType {
 	case ESBulk:
-		requestStr = url + "/_bulk"
+		requestStr = url + "/_bulk"		
```

Comment:
nit: don't add the extra whitespace at the end of the line. You can use `gofmt` or `goimports` to automatically format your code (e.g., `goimports -w pkg/ingest/ingest.go` will fix your imports lines and format the code and then write it back to the file).